std_msgs/Header header #header
yolov7_ros/ObjectData[] objects #list of objects detected in the scene
sensor_msgs/Image source_image #Source image from the camera